bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/api/TapoDeviceConnector.java

```diff
@@ -227,8 +227,8 @@ public void queryInfo(boolean ignoreGap) {
      */
     @Override
```

Comment:
I can't test it but it sounds logical.

Reply:
If you're green with this, can you validate your review ?